internal/services/network/network_packet_capture_resource.go

```diff
@@ -146,6 +157,36 @@ func resourceNetworkPacketCapture() *pluginsdk.Resource {
 					},
 				},
 			},
+
+			"scope": {
```

Comment:
Can we be more explicit here and name this 
```suggestion
			"machine_scope": {
```

Reply:
Updated